dist/assets/config.json

```diff
@@ -1,5 +1,7 @@
 {
-   "baseURL": "https://test.streamstats.usgs.gov/nssservices/",
+   "nssBaseURL": "https://test.streamstats.usgs.gov/nssservices/",
+   "gageBaseURL": "https://test.streamstats.usgs.gov/gagestatsservices/",
```

Comment:
I think to keep terminology consistent/easier to follow, we should call this this `gageStatsBaseURL` instead.